src/ui/components/Avatar.tsx

```diff
@@ -9,8 +9,16 @@ const cx = classNames.bind(css);
 
 // The user image URLs that we get from Google sometimes fail to load, in that case
 // we fall back to a transparent image (instead of showing the browser's icon for broken images)
-export const AvatarImage = (props: React.ImgHTMLAttributes<HTMLImageElement>) => (
-  <img data-private {...props} onError={e => (e.currentTarget.src = "/images/clear.png")} />
+type AvatarImageProps = Omit<React.ImgHTMLAttributes<HTMLImageElement>, "src"> & {
+  src: string | null | undefined;
+};
```

Comment:
> Are you doing this is to account for the fact that the incoming src prop could be null where as the default image props only support string | undefined?

Exactly. I added `null` as a possible value for `User.picture` because that's what we get from GraphQL. And there are several places where `User.picture` is passed to `AvatarImage`, so I went for this instead of converting `null` to `undefined` in several places. Converting from `null` to `undefined` could also be done when receiving the data from GraphQL but, again, there are several places where this would need to be done (and drilling into deeply nested objects just to change `null` to `undefined` also felt weird).

Reply:
Fair enough!